### PR TITLE
[3.0] Add app_name in dashboard

### PR DESCRIPTION
**Pull request by tomswinkels**
We have a few project and every project has horizon. We have place the telescope monitor on a screen with a switch between all the telescopes but we now can't see witch horizon it is.

We think this is a good solution.

### CODE REVIEW

File: resources/views/layout.blade.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <meta name="csrf-token" content="{{ csrf_token() }}">
     <link rel="shortcut icon" href="{{ asset('/vendor/horizon/img/favicon.png') }}">
 
-    <title>Horizon</title>
+    <title>Horizon{{ (config('app.name') ? ' - ' . config('app.name') : '') }}</title>
```
Comment (driesvints) [inline resources/views/layout.blade.php]: You don't need the outer `()` here. 

```suggestion
    <title>Horizon{{ config('app.name') ? ' - ' . config('app.name') : '' }}</title>
```

File: resources/views/layout.blade.php [hunk 2]
```diff
@@ -28,7 +28,7 @@
                 <path class="fill-primary" d="M5.26176342 26.4094389C2.04147988 23.6582233 0 19.5675182 0 15c0-4.1421356 1.67893219-7.89213562 4.39339828-10.60660172C7.10786438 1.67893219 10.8578644 0 15 0c8.2842712 0 15 6.71572875 15 15 0 8.2842712-6.7157288 15-15 15-3.716753 0-7.11777662-1.3517984-9.73823658-3.5905611zM4.03811305 15.9222506C5.70084247 14.4569342 6.87195416 12.5 10 12.5c5 0 5 5 10 5 3.1280454 0 4.2991572-1.9569336 5.961887-3.4222502C25.4934253 8.43417206 20.7645408 4 15 4 8.92486775 4 4 8.92486775 4 15c0 .3105915.01287248.6181765.03811305.9222506z"/>
             </svg>
 
-            <h4 class="mb-0 ml-2"><strong>Laravel</strong> Horizon</h4>
+            <h4 class="mb-0 ml-2"><strong>Laravel</strong> Horizon{{ (config('app.name') ? ' - ' . config('app.name') : '') }}</h4>
```
Comment (driesvints) [inline resources/views/layout.blade.php]:
```suggestion
            <h4 class="mb-0 ml-2"><strong>Laravel</strong> Horizon{{ config('app.name') ? ' - ' . config('app.name') : '' }}</h4>
```

**Comment by tomswinkels** [comment]
Thnx. Maybe also change this at Telescope. https://github.com/laravel/telescope/commit/d9a47f226fabf3ad1603f40f58459a35cccb191e

**Comment by driesvints** [comment]
@tomswinkels just pushed that for Telescope. Thanks for the pr :)

**Pull request merged**